pages/community/precice-workshop-2024.md

```diff
@@ -9,10 +9,47 @@ redirect_from: /preCICE2024/
 
 <img class="img-responsive center-block" src="images/events/precice2024.svg" alt="preCICE Workshop banner" style="max-width: 500px; width: 100%; margin:auto;">
 
-The 5th preCICE Workshop will be held at the [University of Stuttgart](https://www.uni-stuttgart.de/), together with [SimTech](https://www.simtech.uni-stuttgart.de/), on September 24-27, 2024. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Like always, we plan to have user and developer talks, hands-on training sessions, discussions with the developers on your applications and use cases, and plenty of opportunities for networking.
+The 5th preCICE Workshop will be held at the [University of Stuttgart](https://www.uni-stuttgart.de/), together with [SimTech](https://www.simtech.uni-stuttgart.de/), on September 24-27, 2024. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Like always, we plan to have user and developer talks, hands-on training sessions, discussions with the developers on your applications and use cases, and plenty of opportunities for networking. Besides, this year we will organize a preCICE poster session for the first time!
+
+The workshop will start with a course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and non-beginners, since advanced topics will also be covered. The main part of the workshop will take part on September 25-27. A detailed program for the workshop will be provided later on. 
```

Comment:
```suggestion
The workshop will start with a hands-on training course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and current preCICE users, since advanced topics will also be covered. The main part of the workshop will take part on September 25-27. A detailed program for the workshop will be provided later on. 
```